mesh/mesh.cpp

```diff
@@ -6478,6 +6510,52 @@ void Mesh::GetEdgeToUniqueKnotvector(Array<int> &edge_to_ukv,
    }
 }
 
+void Mesh::LoadNonconformingPatchTopo(std::istream &input,
+                                      Array<int> &edge_to_knot)
```

Comment:
```suggestion
                                      Array<int> &edge_to_ukv)
```
Just for consistency with `ncnurbs.cpp` line 38 

```cpp
patchTopo->LoadNonconformingPatchTopo(input, edge_to_ukv);
```